app/scripts/lib/network-store.js

```diff
@@ -31,7 +31,6 @@ export default class ReadOnlyNetworkStore {
       const response = await fetchWithTimeout(FIXTURE_SERVER_URL);
       if (response.ok) {
         this._state = await response.json();
-        this.mostRecentRetrievedState = this._state;
```

Comment:
This is why the test had passed earlier when it should have failed: this property was set too early, before the first read. In production it's only set upon read.

Reply:
Good catch!